include/zfs_namecheck.h

```diff
@@ -43,6 +43,8 @@ typedef enum {
 	NAME_ERR_RESERVED,		/* entire name is reserved */
 	NAME_ERR_DISKLIKE,		/* reserved disk name (c[0-9].*) */
 	NAME_ERR_TOOLONG,		/* name is too long */
+	NAME_ERR_SELF_REFERENCE,	/* reserved self path name ('.') */
+	NAME_ERR_PARNT_REFERENCE,	/* reserved parent path name ('..') */
```

Comment:
Sorry, I missed this initially.  `NAME_ERR_PARNT_REFERENCE` should be `NAME_ERR_PARENT_REFERENCE`.

Reply:
Unfortunately, It crosses that 80 words limit in  `module/zcommon/zfs_namecheck.c` line 250. That why I had to change it to `NAME_ERR_PARNT_REFERENCE`. Should I go for another look around?

Reply:
REF is often used as shorthand for REFERENCE. So try NAME_ERR_PARENT_REF

Reply:
Please use the correct spelling.  You can break line 250 after the `=`.

Reply:
Corrected!